hbase-shaded-netty/pom.xml

```diff
@@ -141,7 +141,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-all</artifactId>
-      <version>4.1.34.Final</version>
+      <version>4.1.41.Final</version>
```

Comment:
4.1.42 came out 3 days ago. Should use it?

Reply:
Let me address as an addendum. Tried pushing here but no perms.